Framework/WorkflowAlgorithms/inc/MantidWorkflowAlgorithms/MuonGroupAsymmetryCalculator.h

```diff
@@ -35,16 +35,19 @@ class DLLExport MuonGroupAsymmetryCalculator : public MuonGroupCalculator {
   MuonGroupAsymmetryCalculator(const API::WorkspaceGroup_sptr inputWS,
                                const std::vector<int> summedPeriods,
                                const std::vector<int> subtractedPeriods,
-                               const int groupIndex);
+                               const int groupIndex, const double start = 0.0,
+                               const double end = 30.0);
   /// Performs group asymmetry calculation
   API::MatrixWorkspace_sptr calculate() const override;
 
 private:
   /// Removes exponential decay from the workspace
   API::MatrixWorkspace_sptr removeExpDecay(const API::Workspace_sptr &inputWS,
                                            const int index) const;
+  API::MatrixWorkspace_sptr
+  estimateAsymmetry(const API::Workspace_sptr &inputWS, const int index) const;
 };
-
+std::vector<double> convertToVec(std::string const &list);
```

Comment:
Is that supposed to be here? There is no `convertToVec` and this is outside the class definition